app/services/search/vacancy_search.rb

```diff
@@ -50,7 +50,7 @@ def vacancies
   end
 
   def total_count
-    vacancies.count
+    @total_count ||= vacancies.count
```

Comment:
Memoise so it avoids triggering the `vacancies` scope query each time we retrieve its size.